Source/WebCore/platform/audio/cocoa/MediaSessionManagerCocoa.mm

```diff
@@ -390,10 +390,11 @@
         auto cfCurrentTime = adoptCF(CFNumberCreate(kCFAllocatorDefault, kCFNumberDoubleType, &nowPlayingInfo.currentTime));
         CFDictionarySetValue(info.get(), kMRMediaRemoteNowPlayingInfoElapsedTime, cfCurrentTime.get());
     }
-    if (nowPlayingInfo.artwork && nowPlayingInfo.artwork->imageData) {
-        auto nsArtwork = nowPlayingInfo.artwork->imageData->makeContiguous()->createNSData();
-        CFDictionarySetValue(info.get(), kMRMediaRemoteNowPlayingInfoArtworkData, nsArtwork.get());
-        CFDictionarySetValue(info.get(), kMRMediaRemoteNowPlayingInfoArtworkMIMEType, nowPlayingInfo.artwork->mimeType.createCFString().get());
+    if (nowPlayingInfo.artwork && nowPlayingInfo.artwork->image && nowPlayingInfo.artwork->image->isBitmapImage() && nowPlayingInfo.artwork->image->tiffRepresentation()) {
+        CFDictionarySetValue(info.get(), kMRMediaRemoteNowPlayingInfoArtworkData, nowPlayingInfo.artwork->image->tiffRepresentation());
```

Comment:
No need to call `nowPlayingInfo.artwork->image->tiffRepresentation()` twice.

Reply:
the first one actually creates the tiff image lazily. The second one return that same image immediately.
I felt this was better than breaking the condition just to store the return value